apollo/src/graphql/model/modelQueries.ts

```diff
@@ -54,8 +58,13 @@ builder.queryFields((t) => ({
                     const [id, dateCreated] = encoder.decode(cursor.cursorToken);
 
                     query
-                        .where('dateCreated', '>=', new Date(dateCreated).toISOString())
-                        .andWhere('id', '>', id);
+                        .where(
```

Comment:
These are the actual changes for review. It looks like everything else is from the pre-commit formatting hooks. I wonder why those weren't being applied before?